config/config.py.template

```diff
@@ -512,6 +516,22 @@ if MLLIB_SPARK_VERSION >= 1.2:
         # (only used for RandomForest)
         OptionSet("feature-subset-strategy", ["auto"])
     ]
+    MLLIB_DECISION_TREE_TEST_OPTS += [
+        # Ensemble type: GradientBoostedTrees
+        OptionSet("ensemble-type", ["GradientBoostedTrees"]),
+        # Path to training dataset (if not given, use random data).
+        OptionSet("training-data", [""]),
```

Comment:
Isn't this appending copies of the same values to MLLIB_DECISION_TREE_TEST_OPTS?  I think you can just updates "ensemble-type" to include GBT as well, and then each will get passed to the test in turn.